Controls/Chart/Chart.UWP/Visualization/CartesianChart/CartesianChartGrid.cs

```diff
@@ -469,6 +480,23 @@ public void Update()
                 }
             }
 
+            public void UpdateVisuals()
+            {
+                if (dashArray == null)
+                {
+                    this.dashArray = new DoubleCollection() { 4, 2};
```

Comment:
We can take it from the Style. The reason I didn't do it was to distinguish the Composition from the Styles used by the visual elements that draw the ChartGrid when Composition is not used. 

It couldn't be overridden, but I have provided an easy way for the users to set the desired dash array through the  PrepareCartesianChartGridLineVisual method - there they can provide any desired dash array.